src/users.py

```diff
@@ -157,6 +157,21 @@ def __iter__(self):
     def items():
         yield from var.USERS.items()
 
+def complete_match(string, users):
+    matches = []
+    string = lower(string)
```

Comment:
`lower` *is* `irc_lower`, except better. The latter is going away after the transition.